hyperspy/external/mpfit/mpfit.py

```diff
@@ -1448,7 +1448,11 @@ def defiter(self, fcn, x, iter, fnorm=None, functkw=None,
 
         # Determine which parameters to print
         nprint = len(x)
-        print("Iter ", ('%6i' % iter), "   CHI-SQUARE = ", ('%.10g' % fnorm), " DOF = ", ('%i' % dof))
+        print(
+            "Iter ", ('%6i' %
+                      iter), "   CHI-SQUARE = ", ('%.10g' %
+                                                  fnorm), " DOF = ", ('%i' %
+                                                                      dof))
```

Comment:
As for the arch instructions, I can get to it in a couple of weeks, unless
someone wants to do it before then. Maybe comment it out for now and open a
new issue about it so we don't forget?

On Tue, Jun 14, 2016 at 8:53 AM, Tomas Ostasevicius <
notifications@github.com> wrote:

> In hyperspy/external/mpfit/mpfit.py
> <https://github.com/hyperspy/hyperspy/pull/841#discussion_r66963260>:
>
> > @@ -1448,7 +1448,11 @@ def defiter(self, fcn, x, iter, fnorm=None, functkw=None,
> >
> >          # Determine which parameters to print
> >          nprint = len(x)
> > -        print("Iter ", ('%6i' % iter), "   CHI-SQUARE = ", ('%.10g' % fnorm), " DOF = ", ('%i' % dof))
> > +        print(
> > +            "Iter ", ('%6i' %
> > +                      iter), "   CHI-SQUARE = ", ('%.10g' %
> > +                                                  fnorm), " DOF = ", ('%i' %
> > +                                                                      dof))
>
> Sure, should be done
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/hyperspy/hyperspy/pull/841/files/ef250ad01582fe4ffa29eb62e0b60ed12e93d4a9#r66963260>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABOBXbmpM0o5FsTf6753PrsRonhQhbdmks5qLqRQgaJpZM4HI3Hl>
> .
>



-- 
Joshua Taillon
Materials Science and Engineering
University of Maryland, College Park
www.joshuataillon.com